File: UNIT 2/C/ases1.c
#include <stdio.h>

int main() {
    int a, b, c;
    printf("Masukkan tiga bilangan: ");
    scanf("%d %d %d", &a, &b, &c);

    int max = a, min = a;

    if (b > max) max = b;
    if (c > max) max = c;
    if (b < min) min = b;
    if (c < min) min = c;

    printf("Bilangan terbesar: %d\n", max);
    printf("Bilangan terkecil: %d\n", min);

    return 0;
}

File: UNIT 2/py/ases1.py
a = int(input("Masukkan bilangan pertama: "))
b = int(input("Masukkan bilangan kedua: "))
c = int(input("Masukkan bilangan ketiga: "))

# Asumsikan a sebagai terbesar dan terkecil
terbesar = a
terkecil = a

# Cek untuk nilai terbesar
if b > terbesar:
    terbesar = b
if c > terbesar:
    terbesar = c

# Cek untuk nilai terkecil
if b < terkecil:
    terkecil = b
if c < terkecil:
    terkecil = c

print("Bilangan terbesar:", terbesar)
print("Bilangan terkecil:", terkecil)
